.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -5,51 +5,75 @@ about: Interested in contributing a new component to Fabric? This template inclu
 ---
 <!-- Use this template for new components or new component varients -->
 
-## Component Details
+## Component details
 
-<!-- Please provide a general description of the component(s) being added -->
+### Description
```

Comment:
Why remove the comment--are you hoping a heading with "Description" will point people in the same direction?

Reply:
It seems somewhat redundant given the comment on line 6.

But now that we're on the subject... varients should be "variants" (I think)